docs/src/build_config/passthrough/hypervisor_options.md

```diff
@@ -0,0 +1,110 @@
+# Microvm hypervisor options
+
+Microvm is the component responsible for launching virtual machines.
+
+### Options definitions
+
+A virtual machine is defined under Ghaf’s subdirectory ``microvmConfigurations/VM_NAME/default.nix``, e.g.:
+
+```
+microvmConfigurations/memshare/default.nix
+https://github.com/jkuro-tii/ghaf/blob/main/microvmConfigurations/memshare/default.nix
+```
+
+This file contains hypervisor’s options for running the VM. For each hypervisor there is a bunch of microvm’s predefined options:
+
+#### crosvm
```

Comment:
I kind of like what you're trying to do here with the hypervisor specific options. Would it work better if we tried contributing the hypervisor-specific supported options to `microvm.nix` documentation as a table instead of lists for `crosvm` and `qemu` here? This is kind of partially duplicating what's already defined here https://astro.github.io/microvm.nix/microvm-options.html and I'm afraid we may risk getting out of sync with the upstream documentation (see your note in the last line of this PR).

Reply:
I replaced the list of options with link to the documentation.